contracts/sol6/mock/MockReserveSanity.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity 0.6.6;
+
+import "../IKyberReserve.sol";
+import "../SanityRatesGasPrice.sol";
+import "../utils/Utils5.sol";
+import "../utils/zeppelin/SafeERC20.sol";
+
+
+contract MockReserveSanity is IKyberReserve, Utils5 {
+    using SafeERC20 for IERC20;
+
+    SanityRatesGasPrice public sanityRatesContract;
+    mapping(address => uint256) public buyTokenRates;
+    mapping(address => uint256) public sellTokenRates;
+
+    function setContracts(
+        SanityRatesGasPrice _sanityRates
+    ) public {
+        sanityRatesContract = _sanityRates;
+    }
+
+    function setRate(
+        IERC20 token,
+        uint256 buyRate,
+        uint256 sellRate
+    ) public {
+        buyTokenRates[address(token)] = buyRate;
+        sellTokenRates[address(token)] = sellRate;
+    }
+
+    function trade(
```

Comment:
This function needs to be overriden?

Reply:
Needed since IKyberReserve is implemented.